packages/main/src/themes/Avatar.css

```diff
@@ -210,4 +211,48 @@
 
 .ui5-avatar-initials {
 	color: inherit;
+}
+
+/* Slotted Badge */
+::slotted([ui5-badge][slot="badge"]) {
```

Comment:
The style `:slotted([ui5-badge][slot="badge"])`
 will apply only if ui5-badge is used, and won't apply to custom badge.
 I think `:slotted([slot="badge"])` is more accurate.
 
 *the comment applies to the whole file

Reply:
Done